pkg/instrumentation/upgrade/upgrade.go

```diff
@@ -74,5 +76,21 @@ func (u *InstrumentationUpgrade) upgrade(_ context.Context, inst v1alpha1.Instru
 			inst.Annotations[v1alpha1.AnnotationDefaultAutoInstrumentationJava] = u.DefaultAutoInstrJava
 		}
 	}
+	autoInstNodeJS := inst.Annotations[v1alpha1.AnnotationDefaultAutoInstrumentationNodeJS]
+	if autoInstNodeJS != "" {
+		// upgrade the image only if the image matches the annotation
+		if inst.Spec.NodeJS.Image == autoInstNodeJS {
+			inst.Spec.NodeJS.Image = u.DefaultAutoInstrNodeJS
+			inst.Annotations[v1alpha1.AnnotationDefaultAutoInstrumentationNodeJS] = u.DefaultAutoInstrNodeJS
+		}
+	}
+	autoInstNodePython := inst.Annotations[v1alpha1.AnnotationDefaultAutoInstrumentationPython]
```

Comment:
`autoInstNodePython` -> `autoInstPython` node as python suffix is bit confusing.

Reply:
good catch this is a typo. fixed